socorro/unittest/cron/jobs/test_clean_missing_symbols.py

```diff
@@ -1,26 +1,17 @@
 import datetime
 
-from crontabber.app import CronTabber
-
+from socorro.cron.crontabber_app import CronTabberApp
 from socorro.lib.datetimeutil import utc_now
 from socorro.unittest.cron.jobs.base import IntegrationTestBase
-from socorro.unittest.cron.setup_configman import (
-    get_config_manager_for_crontabber,
-)
 
 
 class TestCleanMissingSymbolsCronApp(IntegrationTestBase):
 
     def _setup_config_manager(self, days_to_keep=None):
-        super(TestCleanMissingSymbolsCronApp, self)._setup_config_manager
```

Comment:
I'm puzzled as to what the intention of this line was. It evaluates to a function, but doesn't call the function. This is repeated in a bunch of the tests.

I suspect someone started to do something, left it in the commit, it landed, then it was copy/pasted around without anyone looking at it.